### ISSUE TITLE
Add "Set on-chain identity" and "Edit profile" to user menu

### ISSUE THREAD

**Issue by raykyri** [hicommonwealth/commonwealth-oss#183]
We have a user menu now, which shows up in the top right of the page whenever there is an active user. (It is implemented twice, once for when there is an active address, once for without.)

We should add the "Set on-chain identity" and "Edit profile" menu options, to that menu, which bring up the respective modals to do those things. These menu options should only show up if there's an active address that supports the features, i.e. we have an active account on a Substrate chain.

---

Side note, in another PR we can also fetch the old code for displaying wallet balances there and bring that back...but I haven't made that a task for now because I think it is mainly blocked on design and I'd rather just do that in one go.
